### ISSUE TITLE
TRANSLATION_GUIDE.md 용어집 "gradient descent" 단어 추가

### ISSUE THREAD

**Issue by 95hj** [PyTorchKorea/tutorials-kr#555]
## 이슈 내용
_어떤 이슈인지 알려주세요._
용어집에 gradient descent 를 경사하강법으로 추가하는 것이 바람직하다고 생각합니다.

## 추가 정보
_다른 참고할 정보 또는 URL 등이 있다면 알려주세요._
